bedrock/products/templatetags/misc.py

```diff
@@ -19,6 +21,46 @@
 TAX_NOT_INCLUDED = ["US", "CA"]
 
 
+def _vpn_get_ga_data(bundle_relay, plan, selected_plan):
```

Comment:
@alexgibson this feels like the hack-iest hack to ever hack but the "right way" would require totally re-structuring the data we have. At least if this goes horribly wrong it doesn't impact the user's experience "just" our analytics. You probably know the VPN stuff best, is there anything I can do to improve this approach?

Reply:
@stephaniehobson yikes, this is a LOT of string wrangling :/

Honestly I think the easiest and less error prone way to handle this would be to add some new metadata for analytics to our pricing matrix in settings.py. Something like:

```
"12-month": {
    "id": "price_1J0Y1iKb9q6OnNsLXwdOFgDr" if DEV else "price_1Iw85dJNcmPzuWtRyhMDdtM7",
    "price": "US$4.99",
    "total": "US$59.88",
    "saving": 50,
    "analytics": {
        "category": "vpn",
        "currency": "USD",
        "discount": "60.00",
        "price": "59.88",
        "variant": "yearly"
    }
},
```

It's perhaps more effort up front to calculate, but potentially much error prone to trying to manipulate data this way? If we keep everything in a neat and tidy `analytics` object, then we can also avoid a somewhat painful refactor hopefully? 🤞

Reply:
I like this a good middle ground between totally redoing everything and hacking on the strings 💡 Thank you!